lib/extract-resources.ts

```diff
@@ -22,11 +26,13 @@ export async function extractResources(url: string, browser: Browser) {
     return { firstPage, totalPages };
   });
 
+  console.log(totalPages)
+
   await page.mouse.move(500, 500);
 
   let otherPages: string[] = [];
-  for (let i = 0; i < totalPages - 1; i++) {
-    await page.mouse.wheel({ deltaY: 1185 });
+  for (let i = 0; i < totalPages*2; i++) {
```

Comment:
didn't quite understand the scrolling logic. With this settings it should be able to handle very long and very short scores equally well without skipping any pages